nengo_viz/static/viz_netgraph_item.js

```diff
@@ -36,7 +36,7 @@ VIZ.NetGraphItem = function(ng, info) {
     this.minHeight = 5;
 
     this.expanded = false;
-    
+
```

Comment:
`git diff -w` does a diff ignoring whitespace changes, which is useful once you've added all the non-whitespace changes to make sure you've got everything. Unfortunately, there doesn't seem to be anything like `git add -pw`.